frontend/app/templates/components/nav-bar.hbs

```diff
@@ -1,44 +1,43 @@
-<div class="ui top absolute menu">
-    {{#link-to 'index' class="header item"}}
-    <h2 style="font-weight: normal;">badgeyay</h2>
-    {{/link-to}}
-    <div class="right menu">
-        {{#if session.isAuthenticated}}
-            {{#ui-dropdown class='large item' action='hide'}}
-                <div class="upper">
-                    <img class="ui avatar image" src="{{session.currentUser.photoURL}}">
-                    <span class="mobile-hide">{{session.currentUser.displayName}}</span>
-                </div>
-                <i class="dropdown icon"></i>
-                <div class="menu profile-dropdown-menu">
-                    {{#link-to 'my-badges' class="large item navbar-dropdown upper"}}
-                        My Badges
-                    {{/link-to}}
-                    {{#link-to 'settings' class="large item navbar-dropdown upper"}}
-                        Settings
-                    {{/link-to}}
-                    {{#if authToken.permissions.isAdmin}}
-                        {{#link-to 'admin' class="large item navbar-dropdown upper"}}
-                            Admin
-                        {{/link-to}}
-                    {{/if}}
-                    <div class="divider"></div>
-                    <a class="large item navbar-dropdown upper" {{action 'logOut'}}>Logout<i class="right aligned sign out alternate icon pull-right"></i></a>
-                </div>
-            {{/ui-dropdown}}
-        {{else}}
-            {{#link-to 'login'}}
-                <div class="right item ui button" style="text-transform: uppercase; font-weight: 400; line-spacing: 0.5px; line-height: 30px">Login</div>
-            {{/link-to}}
-        {{/if}}
-        {{#if session.isAuthenticated}}
-          {{#link-to 'create-badges' class='item ui orange text create'}}
-                CREATE BADGES
-            {{/link-to}}
-        {{else}}
-            {{#link-to 'login' class='item ui orange text create' click=(action 'loginToContinue')}}
-                CREATE BADGES
-            {{/link-to}}
-        {{/if}}
-    </div>
-</div>
\ No newline at end of file
+<div class="mobile hidden row">
```

Comment:
Please conform with the intendation of 4 spaces used throughout the source code.

Reply:
I just missed a thing , It will hide the complete nav-bar in `Mobile View`

Reply:
I spoke about intendation, anyway if it's WIP pls add the necessary changes to the pr title.

Reply:
I mentioned about intendation, anyway if it's WIP please mark it as WIP.